endpoint.go

```diff
@@ -555,33 +445,6 @@ func (ep *endpoint) Delete() error {
 	return nil
 }
 
-func (ep *endpoint) Statistics() (map[string]*sandbox.InterfaceStatistics, error) {
```

Comment:
IMO the statistics belong to the endpoint. we can provide a convenience API at the sandbox layer for multiple endpoints. But removing it from here doesnt seem right.